build-tools/proguard/proguard.mdproj

```diff
@@ -0,0 +1,12 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
Xamarin Studio is..."finicky"...about project formats.

Please follow https://github.com/xamarin/xamarin-android/blob/6cfe2a3ee8943687fd325c46979f915ba619d274/build-tools/libzip/libzip.mdproj as an example. Specifically, to appease Xamarin Studio, `proguard.mdproj` must have a `<PropertyGroup/>` for each "configuration", Debug and Release.